packages/snap/src/keyring.ts

```diff
@@ -199,7 +199,10 @@ export class SimpleKeyring implements Keyring {
     await this.#saveState();
     return {
       pending: true,
-      redirect: {},
+      redirect: {
+        url: 'http://localhost:8000/',
```

Comment:
Is this correct?  In the production version, don't you want to redirect to https://metamask.github.io/snap-simple-keyring/latest/?

Reply:
Indeed, probably the URL should be in the `.env`.

Reply:
this was only for testing.